### PR TITLE
Add video gege

### PR DESCRIPTION
**Pull request by ShawnCone**
Addressing issue #33

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -193,7 +193,15 @@ <h3>Chapter 2: Education</h3>
                     </div>
                 </section>
             </article>
-
+            
+            <!-- Video Gege -->
+            <section class="wrapper style3 container special" id="our-message">
```
Comment (linahsie) [inline index.html]: Ini masukin di section <--Three--> aja gausa bikin section baru

File: assets/sass/main.css [hunk 2]
```diff
@@ -2228,3 +2228,18 @@ table, th, td {
...
   margin-bottom: 5.5%;}
 
+/* Styles for video gege */
+
+.iframe-container{
+  position: relative;
+  width: 100%;
+  padding-bottom: 56.25%; 
+  height: 0;
+}
```
Comment (linahsie) [inline assets/sass/main.css]: add newline between styling
```{r}
}
  <-- newline
.iframe-container iframe {
```

File: assets/sass/main.css [hunk 3]
```diff
@@ -2228,3 +2228,18 @@ table, th, td {
...
+.iframe-container iframe{
```
Comment (linahsie) [inline assets/sass/main.css]: space between class name and {

**Review by linahsie — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ShawnCone) [reply index.html]: updated in the newest commit

**Comment by ShawnCone** [comment]
@linahsie since fix_chap_1_responsiveness is done, duplicate lines for the same iframe container style is removed here.

**Review by linahsie — APPROVED** [review]

**Pull request merged**